src/commands/show.ts

```diff
@@ -49,9 +51,36 @@ async function showProblemInternal(id: string): Promise<void> {
             return;
         }
 
-        const outDir: string = await selectWorkspaceFolder();
+        let outDir: string = await selectWorkspaceFolder();
+        const outputPathCfg: string = leetCodeConfig.get<string>("outputPath") || "";
+        const outputPath: RegExpMatchArray | null = outputPathCfg.match(/\$\{(.*?)\}/);
+        if (outputPath) {
+            switch (outputPath[1].toLowerCase()) {
+                case "tag": {
+                    const closestTag: string = node.tags.reduce((prev: string, curr: string) => {
```

Comment:
Hmm, here is the problem, looks like you want to find out the tag with the longest length if the problem has multiple tags. Any reason for this logic?

Reply:
I found that if I took the first one, most of them are [array]. This is not what I want. By observing the all of tags, I found that the longest tag is generally the best one...I originally wanted to add a priority to the tag, which is complicated.

Reply:
Then, this is not convincing enough. Please considering:
- We have no idea what does user want
- There is no data to support the conclusion: `the longest tag is generally the best one`, it's just a subjective judgment

Meanwhile, by doing this, we provided an implicit logic for the user, this is not a recommended user experience.

I have two suggestions:
1. show quick picks to the user if multiple tags are available, let the user choose one.
2. Remove the support for `${tag}` for now.

Reply:
Thanks for your suggestion，This solution is very reasonable